helpers/api.js

```diff
@@ -1,4 +1,5 @@
 exports.getBaseUrl = () => {
-  const env = process.env.DDENV === "prod" ? "" : "_" + process.env.DDENV;
+  // If DDENV is defined, use it, otherwise default to prod
+  const env = process.env.DDENV ? "_" + process.env.DDENV : "";
```

Comment:
We should make this work with prod set explicitly too (as this would be the only environment where that wouldn't work)
This was originally a two step thing, converted to process.env would look like
```javascript
 const env = process.env.DDENV || "prod";  // default to prod
  const envUrlPart = env === "prod" ? "" : "_" + env;
// use envUrlPart in format string
```

Reply:
@mikedeploysdrones Done